### PR TITLE
A couple of documentation tweaks

### PR DESCRIPTION
**Pull request by davehunt**

### CODE REVIEW

File: docs/tests-ui.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ Installing Dependencies
 
 3. From the `tests/ui/` directory, use NPM or another package manager to install Intern::
 
+    mkdir node_modules
```
Comment (groovecoder) [inline docs/tests-ui.rst]: Weird. I didn't have to do this. :confused:

**Comment by groovecoder** [comment]
Thanks @davehunt. Per [our contributing guidelines](https://github.com/mozilla/kuma/blob/master/CONTRIBUTING.md), can you prefix your commit messages with a bug number? For these kinds of docs/setup issues, I always (re-)use [Bug 957802 - Streamline new contributor setup](https://bugzilla.mozilla.org/show_bug.cgi?id=957802). E.g.:
```
bug 957802 - Fix Intern link
bug 957802 - Create node_modules directory to force local package install
```

File: docs/tests-ui.rst (hunk 1 above)
Comment (darkwing) [inline docs/tests-ui.rst]: I'd prefer we left that out; `npm` is meant to create the directory if it's not there so you may need to configure your specific machine differently.

File: docs/tests-ui.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Client-side Testing with Intern
 ===============================
 
-The client-side testing tool for the MDN front-end is `Intern <http://intern.io>`_, a Selenium WebDriver API which allows developers to write automated testing via JavaScript. Intern is an open source project created and maintained by `SitePen <http://sitepen.com>`_.
+The client-side testing tool for the MDN front-end is `Intern https://theintern.github.io/>`_, a Selenium WebDriver API which allows developers to write automated testing via JavaScript. Intern is an open source project created and maintained by `SitePen <http://sitepen.com>`_.
```
Comment (darkwing) [inline docs/tests-ui.rst]: Good catch, they moved the site since I started tests! :)

File: docs/tests-ui.rst (hunk 1 above)
Comment (davehunt) [inline docs/tests-ui.rst]: Okay, I wonder why it wasn't created on my machine then.

**Comment by davehunt** [comment]
Updated.

File: docs/tests-ui.rst (hunk 2 above)
Comment (darkwing) [inline docs/tests-ui.rst]: Ooops, I think you're missing a `<` at the beginning of the link?

Comment (davehunt) [inline docs/tests-ui.rst]: Gah! Well spotted. Fixed now.

**Comment by darkwing** [comment]
R+, thank you!

**Pull request merged**